src/backend/tsconfig.json

```
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "checkJs": false,
+    "noEmit": true,
+    "typeRoots": ["types", "node_modules/@types"]
+  },
+  "include": ["*.js", "test/**/*"],
```

```suggestion
  "include": ["**/*.js"],
```
I think we want this to include all js files in `src/backend` right? Especially since it's opt-in via `// @ts-check`. Can drop the `test` option, since that's included in the first now.